server/admin/models.py

```
@@ -43,4 +43,19 @@ class DomainChoice(models.TextChoices):
 class Room(models.Model):
     uid = models.UUIDField(default=uuid.uuid4, primary_key=True)
     curr_player = models.ForeignKey(to=Player, null=True, on_delete=models.SET_NULL)
-    name = models.CharField(max_length=25)
\ No newline at end of file
+    name = models.CharField(max_length=25)
+
+from django.db import models
+from admin.models import Room
```

Ruff (0.15.0)

[error] Redefinition of unused `Room` from line 43: `Room` redefined here

(F811)